impeller/renderer/backend/gles/reactor_gles.cc

```diff
@@ -386,15 +387,22 @@ void ReactorGLES::SetupDebugGroups() {
 
 void ReactorGLES::SetDebugLabel(const HandleGLES& handle,
                                 std::string_view label) {
+  FML_DCHECK(handle.GetType() != HandleType::kFence);
   if (!can_set_debug_labels_) {
     return;
   }
   if (handle.IsDead()) {
     return;
   }
   WriterLock handles_lock(handles_mutex_);
-  if (auto found = handles_.find(handle); found != handles_.end()) {
-    found->second.pending_debug_label = label;
+  if (handle.untracked_id_.has_value()) {
+    handles_to_name_.emplace_back(
+        std::make_tuple(ToDebugResourceType(handle.GetType()),
+                        handle.untracked_id_.value(), std::string(label)));
+  } else {
+    if (auto found = handles_.find(handle); found != handles_.end()) {
+      found->second.pending_debug_label = label;
```

Comment:
We can't just store this in handles_to_name_ right now since the thing might not have been created yet.

Reply:
If the untracked handles are created immediately, why do we need to go through the reactor managed labeling at all? Can we use a different labeling function that calls the debug label methods directly?

Reply:
We can and I originally implemented it that way.  I was thinking these would just get called once in a while so we might as well keep the same semantics, but there are things that we create each frame isn't there?  So maybe I should switch it back.

Reply:
We do a lot of a labeling. it should be mostly switched off in release mode but it will still get in the way for profile mode.

Reply:
done

Reply:
FWIW I'm not sure where we will land on this.  I am seeing code trying to create DeviceBufferGLES from threads that can't react.  That's weird to me since we store ops based on the thread.  I'm still sorting through this.  This is good for now though.